gobblin-temporal/src/main/java/org/apache/gobblin/temporal/yarn/YarnService.java

```diff
@@ -205,11 +206,10 @@ public YarnService(Config config, String applicationName, String applicationId,
 
     this.eventBus = eventBus;
 
-    this.gobblinMetrics = config.getBoolean(ConfigurationKeys.METRICS_ENABLED_KEY) ?
-        Optional.of(buildGobblinMetrics()) : Optional.<GobblinMetrics>absent();
-
-    this.eventSubmitter = config.getBoolean(ConfigurationKeys.METRICS_ENABLED_KEY) ?
-        Optional.of(buildEventSubmitter()) : Optional.<EventSubmitter>absent();
+    // Gobblin metrics have been disabled in the PoC temporarily to make testing kafka integration without having to worry
+    // how the metrics topics are used. Kafka will only be used for GobblinTrackingEvents in the initial POC
```

Comment:
let's steer away from PoC, which maintainers and those considering reuse may not have context on.  instead speak to underlying behavior/semantics (and perhaps suggest when desireable)